manifests/profile.pp

```diff
@@ -257,9 +257,16 @@
     ensure  => present,
   }
 
+  if versioncmp($duplicity::params::duply_version, '1.7.1') < 0 {
```

Comment:
The version is currently based on the operating system only. That is not enough. For instance I can easily install the latest version 1.9.1 on Debian 7 (via the archive) and yet the version check would think it is 1.5.5.5.

I suggest to add block like the following to the `init.pp` which does the distinction between the archive and the regular package and uses the right version

```
  $real_duply_version = empty($duply_version) ? {
    true => $duply_package_provider ? {
      archive => $duply_archive_version,
      default => $duplicity::params::duply_version,  # fall back to the default ...
    },
    default   => $duply_version,  # in case the default package version is wrong
  }
```

This would also allow to give the module a hint in case the platform is not supported. What do you think?

Other than that it looks pretty good!